src/com/facebook/buck/cli/QuickstartCommand.java

```diff
@@ -71,6 +77,19 @@ public String getDestDir() {
   }
 
   public String getAndroidSdkDir() {
+    if (androidSdkDir == null) {
+      ProjectFilesystem projectFilesystem = new ProjectFilesystem(Paths.get("."));
+
+      AndroidDirectoryResolver androidDirectoryResolver =
+          new DefaultAndroidDirectoryResolver(projectFilesystem,
+              Optional.<String>absent(),
+              new DefaultPropertyFinder(projectFilesystem, ImmutableMap.copyOf(System.getenv())));
```

Comment:
I am afraid that when we run with the daemon, `System.getenv()` will give you the environment of the daemon process. We should make this use the client's environment.